interface/CorrectedMuonProducerT.h

```diff
@@ -23,39 +23,13 @@
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
 
 #include <cp3_llbb/Framework/interface/Rochester.h>
-#include <KaMuCa/Calibration/interface/KalmanMuonCalibrator.h>
```

Comment:
Does it give problems, or do you simply not need KaMuCa? If it's the latter you could also just leave it in without using it (in the recipe and here), the C++ doesn't diverge so much then

Reply:
It wouldn't compile under CMSSW_9_2_6p2, that's why I removed it.